plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -1213,6 +1226,104 @@ public function backfill( array $args = array(), array $assoc_args = array() ) {
 		);
 	}
 
+	/**
+	 * Show the list of WooCommerce-aware plugins known to be compatible, incompatible or without compatibility declaration for HPOS. Note that inactive plugins will always be listed in the "uncertain" list.
+	 *
+	 * [--include-inactive]
+	 * : Include inactive plugins in the list.
+	 *
+	 * [--display-filenames]
+	 * : Print plugin file names instead of plugin names.
+	 *
+	 * @since 9.1.0
+	 *
+	 * @param array $args       Positional arguments passed to the command.
+	 * @param array $assoc_args Associative arguments (options) passed to the command.
+	 */
+	public function compatibility_info( array $args = array(), array $assoc_args = array() ) {
+		$container          = wc_get_container();
+		$feature_controller = $container->get( FeaturesController::class );
+		$plugin_info        = $feature_controller->get_compatible_plugins_for_feature( 'custom_order_tables', ! ( (bool) ( $assoc_args['include-inactive'] ?? null ) ) );
+		$display_filenames  = (bool) ( $assoc_args['display-filenames'] ?? null );
+
+		$compatibles       = $this->get_printable_plugin_names( $plugin_info['compatible'], $display_filenames );
+		$compatibles_count = count( $compatibles );
+		$this->log(
+			sprintf(
+				// translators: $1$d = plugins count, %2$s = colon (if list follows) or empty.
+				__( '%%C%1$d%%n compatible plugins found%2$s', 'woocommerce' ),
```

Comment:
Here and for similar lines, since there could be zero, one or many plugins found, should we either swap `plugins` for `plugin(s)` or else use [_n()](https://developer.wordpress.org/reference/functions/_n/) (I believe this is the most internationally friendly approach)?